### DISCUSSION TITLE
Should `airflow dags test` support more use cases, eg, passing a `conf` param?

### DISCUSSION THREAD

**Discussion by jeffkpayne** [apache/airflow#17522] [Ideas]
We're currently migrating to AF 2.x and found the local `airflow dags test` testing approach to be extremely useful.  We also leverage ad-hoc/unscheduled DAGs that rely on run-time configs.  However, that command currently doesn't support passing a `conf` param.  Does it make sense to add support for that and other use cases?  Would others find that useful?

FWIW, looking at https://github.com/apache/airflow/blob/main/airflow/cli/commands/dag_command.py#L392, the change to support passing a conf looks pretty trivial. I'm more than willing to open a PR, if this seems useful.

**Comment by potiuk** [comment]
Please open PR :)
